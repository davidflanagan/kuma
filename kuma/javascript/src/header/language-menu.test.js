//@flow
import React from 'react';
import { create } from 'react-test-renderer';
import DocumentProvider from '../document-provider.jsx';
import { fakeDocumentData } from '../document-provider.test.js';
import LanguageMenu from './language-menu.jsx';

test('LanguageMenu snapshot', () => {
    const page = create(
        <DocumentProvider initialDocumentData={fakeDocumentData}>
            <LanguageMenu />
        </DocumentProvider>
    );
    expect(page.toJSON()).toMatchSnapshot();
});
