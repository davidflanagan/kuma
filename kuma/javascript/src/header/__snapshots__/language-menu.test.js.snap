// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageMenu snapshot 1`] = `
.emotion-7 {
  position: relative;
  pointer-events: auto;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  padding: 0 5px;
  margin: 0;
  border: none;
}

.emotion-3:hover {
  background-color: #eee;
}

.emotion-3:focus {
  outline-offset: -3px;
}

.emotion-0 {
  height: 1.5em;
}

.emotion-1 {
  font-size: '75%';
  padding-left: 2px;
}

.emotion-5 {
  position: absolute;
  z-index: 100;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-sizing: border-box;
  background-color: white;
  border: solid #83d0f2 1.5px;
  box-shadow: 3px 3px 5px rgba(0,0,0,0.25);
  padding: 10px;
  min-width: 100%;
  display: none;
}

.emotion-5 li {
  padding: 5px;
  white-space: nowrap;
}

<div
  className="emotion-7 emotion-8"
>
  <button
    className="emotion-3 emotion-4"
    onClick={[Function]}
  >
    <svg mock
      className="emotion-0"
    />
    <span
      className="emotion-1 emotion-2"
    >
      ▼
    </span>
  </button>
  <ul
    className="dropdown-menu emotion-5 emotion-6"
  >
    <li
      lang="es"
    >
      <a
        href="[fake spanish url]"
        title="Spanish"
      >
        <bdi>
          Español
        </bdi>
      </a>
    </li>
  </ul>
</div>
`;
